### PR TITLE
FF116 Relnote: Support for CSP script-src with hashes to allow external files

### PR DESCRIPTION
**Pull request by hamishwillee**
This is release note for supporting CSP script-src using allow list of hashes for external files. It links to heading that will be added by https://github.com/mdn/content/pull/27876

Other docs work can be tracked in #27749

### CODE REVIEW

**Comment by bsmth** [comment]
Taking a look

File: files/en-us/mozilla/firefox/releases/116/index.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ This article provides information about the changes in Firefox 116 that affect d
 
 ### HTTP
 
+- The [Content-Security-Policy](/en-US/docs/Web/HTTP/CSP) now allows specifying [external JavaScript files to be whitelisted using hashes](/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#whitelisting_external_scripts_using_hashes), where previously only inline scripts could be whitelisted using a hash ([Firefox bug 1409200](https://bugzil.la/1409200)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/116/index.md:33]: Minor phrasing one: it reads like we might be missing a word in "The CSP now allows", what do you think of "Configuring a CSP"?

```suggestion
- Configuring a [Content-Security-Policy](/en-US/docs/Web/HTTP/CSP) now supports specifying [external JavaScript files to be whitelisted using hashes](/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#whitelisting_external_scripts_using_hashes), where previously only inline scripts could be whitelisted using a hash ([Firefox bug 1409200](https://bugzil.la/1409200)).
```

**Review by bsmth — APPROVED** [review]
Thanks Hamish, looking great pending one suggestion for your consideration! 👍🏻

File: files/en-us/mozilla/firefox/releases/116/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/116/index.md:33]: Thanks. Yes, is better.

**Comment by bsmth** [comment]
Thanks, Hamish - merging now 👍🏻

**Comment by hamishwillee** [comment]
Thanks!

**Pull request merged**